tests/integration/integration_test.go

```diff
@@ -202,6 +202,64 @@ func TestRunResults(t *testing.T) {
 			assert.Fail(t, fmt.Sprintf("Number of filtered lines is %v, should be %v", len(filtered), 2))
 		}
 	})
+
+	t.Run("Test terraform-aws-bridgecrew-read-only tagging specified tags", func(t *testing.T) {
+		repoPath := utils.CloneRepo("https://github.com/bridgecrewio/terraform-aws-bridgecrew-read-only.git", "a8686215642fd47a38bf8615d91d0d40630ab989")
+		defer func() {
+			_ = os.RemoveAll(repoPath)
+		}()
```

Comment:
any specific reason why to put remove all in a function instead of just directly deferring it `defer os.RemoveAll(repoPath)`

Reply:
Good catch, it used to contain multiple commands, no need for the func if there's only one command to perform